bulk/repo-access-api.adoc

```diff
@@ -119,6 +126,9 @@ We always process one node in its entirety, i.e. we cannot update parts of the n
 A node id referenced as parent, containment, reference, or annotation _can_ be mentioned in the same request, but _can_ be omitted if a node with that id already exists in the repository.
 This way, we can move subtrees and add arbitrary references without sending unchanged nodes.
 
+[horizontal]
```

Comment:
Not as parent, because that would break parent/containment symmetry (and this node would be contained in a non-existing node, thus not be part of a model tree). The others to some degree yes, as we support invalid models.
I can definitely see that for references. Would need to think about the implications of dead containments/annotations.